vin/app/models/concerns/wine.rb

```diff
@@ -1,15 +0,0 @@
-#enconding: utf-8
-
-class Wine < ActiveRecord::Base
```

Comment:
Yeah, you should probably delete the whole `concerns` folder. Maybe use it for the `serializable_hash` method, but nothing else